Entities/Scene.h

```diff
@@ -771,16 +771,15 @@ EntityAllocation(Scene)
 
 
 //////////////////////////////////////////////////////////////////////////////////////////
-// Method:          RetrieveActorsAndDevices
+// Method:          RetrieveSceneObjects
 //////////////////////////////////////////////////////////////////////////////////////////
 // Description:     Sucks up all the Actors and Devices currently active in MovableMan and
 //                  puts them into this' list of objects to place on next load.
-//                  Should be done AFTER RetrieveResidentBrains!
 // Arguments:       The team to only retrieve Actors of. If NoTeam, then all will be grabbed.
 //                  Whether to not get any brains at all.
 // Return value:    How many objects were found knocking about in the world, and stored.
 
-    int RetrieveActorsAndDevices(int onlyTeam = -1, bool noBrains = false);
+    int RetrieveSceneObjects(bool transferOwnership = true, int onlyTeam = -1, bool noBrains = false);
```

Comment:
I believe I had a good reason for putting the new parameter at the start, at the time. I'm not so sure now!

I think that maybe I meant to not give it a default value (as the user of the function should be super careful about it), but didn't get around to it. I'll change that and stop defaulting it to true.